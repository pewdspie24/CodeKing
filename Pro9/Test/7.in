10
1104498912 2114346312
3674510178 4860638495
2486306016 4827110900
1094237998 2270241728
3926030238 5932864088
554606004 2598377016
3043440470 3644189068
1186138561 1998256339
654418541 3963440603
2447175563 2785887721
