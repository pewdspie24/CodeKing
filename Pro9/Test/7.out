11335661
11091806
22920770
13847747
15665452
22458797
6893762
8297777
34670929
2844423
